resources/qml/dialogs/ImagePackSettingsDialog.qml

```diff
@@ -189,16 +193,29 @@ ApplicationWindow {
 
                     }
 
-                    Button {
+                    RowLayout {
                         Layout.alignment: Qt.AlignHCenter
-                        text: qsTr("Edit")
-                        enabled: currentPack.canEdit
-                        onClicked: {
-                            var dialog = packEditor.createObject(timelineRoot, {
-                                "imagePack": currentPack
-                            });
-                            dialog.show();
-                            timelineRoot.destroyOnClose(dialog);
+
+                        Button {
+                            text: qsTr("Edit")
+                            enabled: currentPack.canEdit
+                            onClicked: {
+                                var dialog = packEditor.createObject(timelineRoot, {
+                                    "imagePack": currentPack
+                                });
+                                dialog.show();
+                                timelineRoot.destroyOnClose(dialog);
+                            }
+                        }
+                        Button {
+                            text: qsTr("Remove")
+                            enabled: currentPack.canEdit
+                            onClicked: {
+                                var dialog = packDeleteDialog.createObject(timelineRoot, {
+                                    "imagePack": currentPack
+                                });
+                                dialog.open();
```

Comment:
I think this is missing the destroyOnClose line, which would cause a memory leak.

Reply:
mmh, i'll have a look, because during my first try it said that destroyOnClose didn't exist on dialog.

Reply:
okay, as i expected, when i call `dialog.destroyOnClose()`, i get this:
```
[2023-12-19 16:09:09.452] [qml] [warning] qrc:/resources/qml/dialogs/ImagePackSettingsDialog.qml:218: TypeError: Property 'destroyOnClose' of object ImagePackDeleteDialog_QMLTYPE_222(0x55d1bf224e60) is not a function (qrc:/resources/qml/dialogs/ImagePackSettingsDialog.qml:218, )
```

So there is no `destroyOnClose` on the dialog ^^

Reply:
nvm, me dumb. it's fixed now ^^